FirebaseAuthUI/FUIAuth.h

```diff
@@ -201,6 +201,12 @@ __attribute__((deprecated("Instead use authUI:didSignInWithAuthDataResult:error:
  */
 @property(nonatomic, copy, nullable) NSURL *TOSURL;
 
+/** @property shouldAutoUpgradeAnonymousUsers
+    @brief Whether to enable auto upgrading of anonymous accounts, defaults to NO.
+ */
+@property(nonatomic, assign, getter=shouldAutoUpgradeAnonymousUsers) BOOL
```

Comment:
why the getter is needed?

Reply:
This is a public API, the developer may want to condition on whether shouldAutoUpgrade is turned on or not.

Reply:
@protocol86    I mean the property name and the getter name are identical (shouldAutoUpgradeAnonymousUsers). why getter=?